mscore/musescore.cpp

```diff
@@ -2967,6 +2967,12 @@ AboutBoxDialog::AboutBoxDialog()
       setWindowFlags(this->windowFlags() & ~Qt::WindowContextHelpButtonHint);
 
       copyRevisionButton->setIcon(*icons[int(Icons::copy_ICON)]);
+
+      copyrightLabel->setText(QString("<span style=\"font-size:10pt;\">%1</span>")
```

Comment:
I would go further and not include any HTML marking in the translation. http://musescore.org will redirect to the right language.
The BR can be hardcoded.
The donation page doesn't see to the language selection automatically, we could ask Thomas to do something about it.